dnf-behave-tests/dnf/list.feature

```diff
@@ -54,6 +54,18 @@ Given I drop repository "dnf-ci-fedora"
   setup.noarch +2.12.1-1.fc29 +dnf-ci-fedora
   """
 
+@dnf5
+Scenario: dnf list setup (when setup is installed) using incorrect case
```

Comment:
Could you rename the Scenario to "dnf list is case insensitive"? Your current text only paraphrases what's written in the next line without providing a motivation for the test.

Reply:
Good suggestion. Thank you. Fixed